sdks/python/apache_beam/runners/worker/operations.pxd

```diff
@@ -92,7 +92,7 @@ cdef class DoOperation(Operation):
   cdef DoFnRunner dofn_runner
   cdef object tagged_receivers
   cdef object side_input_maps
-  cdef object user_state_context
+  cpdef public object user_state_context
```

Comment:
Rather than making this public, I would add an `add_timer_info` method to this operation.